scripts/variantstore/wdl/GvsJointVariantCalling.wdl

```diff
@@ -1,27 +1,47 @@
 version 1.0
 
-import "GvsUnified.wdl" as GvsUnified
+import "GvsBulkIngestGenomes.wdl" as BulkIngestGenomes
+import "GvsPopulateAltAllele.wdl" as PopulateAltAllele
+import "GvsCreateFilterSet.wdl" as CreateFilterSet
+import "GvsPrepareRangesCallset.wdl" as PrepareRangesCallset
+import "GvsExtractCallset.wdl" as ExtractCallset
 
 workflow GvsJointVariantCalling {
     input {
         String dataset_name
         String project_id
-        Array[String] external_sample_names
-        Array[File] input_vcfs
-        Array[File] input_vcf_indexes
         String call_set_identifier
         String? extract_output_gcs_dir
         String drop_state = "FORTY"
         Boolean use_classic_VQSR = true
         # Beta users have accounts with tighter quotas, and we must work around that
         Boolean tighter_gcp_quotas = true
+        # TODO should these all be exposed in this WDL?
+        String? sample_id_column_name ## Note that a column WILL exist that is the <entity>_id from the table name. However, some users will want to specify an alternate column for the sample_name during ingest
+        String? vcf_files_column_name
+        String? vcf_index_files_column_name
+        String? sample_set_name ## NOTE: currently we only allow the loading of one sample set at a time
+
+        # This is the most updated snapshot of the code as of July 2, 2023
+        File gatk_override = "gs://gvs_quickstart_storage/jars/gatk-package-4.2.0.0-724-gf478fb2-SNAPSHOT-local.jar"
```

Comment:
do we want to use the more recent gs://gvs_quickstart_storage/jars/gatk-package-4.2.0.0-727-g950da0b-SNAPSHOT-local.jar (July 12, 2023)?

Reply:
mmm actually I'm going to need to bake a fresh one based that includes everything in the July 12 one PLUS the curation script that now stows away in GATK

Reply:
🤦🏻‍♀️ ugh, sorry